### PR TITLE
cargo-*: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
cargo 1.67.1 (8ecd4f20a 2023-01-10)

### CODE REVIEW

File: pages.pt_BR/common/cargo-clippy.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# cargo clippy
+
+> Um conjunto de lints para identificar erros comuns e melhorar o seu código Rust.
+> Mais informações: <https://github.com/rust-lang/rust-clippy>.
+
+- Executar verificações no código no diretório atual:
+
+`cargo clippy`
+
+- Exigir que o `Cargo.lock` esteja atualizado:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-clippy.md:10]:
```suggestion
- Executar verificações garantindo que o `Cargo.lock` esteja atualizado:
```

File: pages.pt_BR/common/cargo-clippy.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Executar verificações em todos os pacotes no workspace:
+
+`cargo clippy --workspace`
+
+- Executar verificações para um pacote específico:
+
+`cargo clippy --package {{pacote}}`
+
+- Tratar avisos como erros:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-clippy.md:22]:
```suggestion
- Executar validações tratando avisos como erros:
```

File: pages.pt_BR/common/cargo-doc.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# cargo doc
+
+> Construir e visualizar a documentação de um pacote Rust offline.
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-doc.md:3]:
```suggestion
> Constrói e visualiza a documentação de um pacote Rust, opcionalmente em modo offline.
```

File: pages.pt_BR/common/cargo-doc.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Mais informações: <https://doc.rust-lang.org/cargo/commands/cargo-doc.html>.
+
+- Construir e visualizar a documentação padrão do pacote no navegador:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-doc.md:6]:
```suggestion
- Construir a documentação padrão do pacote e mostrá-la no navegador:
```

File: pages.pt_BR/common/cargo-doc.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Construir a documentação sem acessar a rede:
+
+`cargo doc --offline`
+
+- Visualizar a documentação de um pacote específico:
+
+`cargo doc --open --package {{pacote}}`
+
+- Visualizar a documentação de um pacote específico offline:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-doc.md:18]:
```suggestion
- Visualizar a documentação de um pacote específico sem acessar a rede:
```

File: pages.pt_BR/common/cargo-rustc.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# cargo rustc
+
+> Compilar um pacote Rust e passar opções extras para o compilador.
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-rustc.md:3]:
```suggestion
> Compila um pacote Rust.
```

Não sei se vale a pena mencionar as opções extra na descrição principal, mas estou disposto a ser convencido do contrário :)

File: pages.pt_BR/common/cargo-rustc.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+> Mais informações: <https://doc.rust-lang.org/cargo/commands/cargo-rustc.html>.
+
+- Compilar o pacote ou pacotes definidos pelo arquivo `Cargo.toml` no diretório de trabalho atual:
+
+`cargo rustc`
+
+- Compilar os artefatos em modo de lançamento (release), com otimizações:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-rustc.md:10]: Como mencionado acima, se calhar "publicação" funciona melhor que "lançamento".

File: pages.pt_BR/common/cargo-rustc.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Compilar com otimizações específicas para a arquitetura do CPU atual:
+
+`cargo rustc --release -- -C target-cpu=native`
+
+- Compilar com otimização de velocidade:
+
+`cargo rustc -- -C opt-level {{1|2|3}}`
+
+- Compilar com otimização de tamanho (`z` também desativa a vetorização de loops):
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-rustc.md:22]: Podemos usar "ciclos" em vez de "loops"?

File: pages.pt_BR/common/cargo-rustc.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo rustc -- -C opt-level {{s|z}}`
+
+- Verificar se o seu pacote usa código inseguro (unsafe):
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-rustc.md:26]:
```suggestion
- Verificar se o pacote usa código com padrões inseguros de acesso à memória:
```

File: pages.pt_BR/common/cargo-test.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# cargo test
+
+> Executar os testes unitários e de integração de um pacote Rust.
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-test.md:3]:
```suggestion
> Executa os testes unitários e de integração de um pacote Rust.
```

File: pages.pt_BR/common/cargo-test.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+> Mais informações: <https://doc.rust-lang.org/cargo/commands/cargo-test.html>.
+
+- Executar apenas os testes que contenham uma string específica em seus nomes:
+
+`cargo test {{nomedoteste}}`
+
+- Definir o número de casos de teste em execução simultânea:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-test.md:10]:
```suggestion
- Definir o número de casos de teste para execução simultânea:
```

File: pages.pt_BR/common/cargo-test.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+`cargo test -- --test-threads={{quantidade}}`
+
+- Exigir que o `Cargo.lock` esteja atualizado:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-test.md:14]:
```suggestion
- Executar os testes garantindo que o `Cargo.lock` esteja atualizado:
```

File: pages.pt_BR/common/cargo-test.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+`cargo test --locked`
+
+- Testar os artefatos em modo de lançamento (release), com otimizações:
```
Comment (waldyrious) [inline pages.pt_BR/common/cargo-test.md:18]: Novamente, sugiro "publicação".

**Review by waldyrious — CHANGES_REQUESTED** [review]
@isaacvicente excellent work, thank you! :clap:

I've left some comments, but feel free to push back on any of them in case you disagree. Cheers, and apologies for the delay in getting to this PR (I'll try to handle the other ones ASAP).

File: pages.pt_BR/common/cargo-build.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+- Compilar um pacote específico:
+
+`cargo build --package {{pacote}}`
+
+- Compilar apenas o binário especificado:
+
+`cargo build --bin {{nome}}`
+
+- Compilar apenas o teste específico:
```
[the remark this answers is not in the document]
Comment (isaacvicente) [reply pages.pt_BR/common/cargo-build.md:30]: Acho que "um teste específico" fica mais simples e claro.

File: pages.pt_BR/common/cargo-build.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
+# cargo build
+
+> Compilar um pacote local e todas as suas dependências.
+> Mais informações: <https://doc.rust-lang.org/cargo/commands/cargo-build.html>.
+
+- Compilar o pacote ou pacotes definidos pelo arquivo `Cargo.toml` no diretório local:
+
+`cargo build`
+
+- Compilar os artefatos em modo de lançamento (release), com otimizações:
```
[the remark this answers is not in the document]
Comment (isaacvicente) [reply pages.pt_BR/common/cargo-build.md:10]: Mudei para "publicação".

File: pages.pt_BR/common/cargo-rustc.md (hunk 7 above)
Comment (isaacvicente) [reply pages.pt_BR/common/cargo-rustc.md:10]: Feito.

File: pages.pt_BR/common/cargo-rustc.md (hunk 8 above)
Comment (isaacvicente) [reply pages.pt_BR/common/cargo-rustc.md:22]: Sim, tudo bem por mim.

File: pages.pt_BR/common/cargo-test.md (hunk 13 above)
Comment (isaacvicente) [reply pages.pt_BR/common/cargo-test.md:18]: Feito.

**Comment by isaacvicente** [comment]
Also, @waldyrious, you don't need to apologize, we're all volunteers :)

**Review by waldyrious — APPROVED** [review]
Thanks for the quick turnaround! Looks good to me :)

**Comment by waldyrious** [comment]
@kbdharun sorry to ping you but I'm not really sure who are the most active maintainers these days. This PR is ready to go as far as I'm concerned, and just needs an extra approving review.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
> This PR is ready to go as far as I'm concerned, and just needs an extra approving review.

Done

**Pull request merged**